src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -240,50 +240,45 @@ double Camera::extract_f_stop() const
 }
 
 void Camera::extract_focal_distance(
-    bool&                   autofocus_enabled,
+    const bool&             autofocus_enabled,
     Vector2d&               autofocus_target,
     double&                 focal_distance) const
 {
     const Vector2d DefaultAFTarget(0.5);        // in NDC
     const double DefaultFocalDistance = 1.0;    // in meters
 
-    if (has_param("focal_distance"))
+    if (autofocus_enabled)
     {
-        if (has_param("autofocus_target"))
+        if (!has_param("autofocus_target"))
```

Comment:
While this is completely correct, I'd find it even easier to read if the `if` clause was reversed.